packages/xpetra/src/CrsGraph/Xpetra_TpetraCrsGraph_decl.hpp

```diff
@@ -326,8 +326,12 @@ namespace Xpetra {
     void getLocalRowView(LocalOrdinal LocalRow, ArrayView< const LocalOrdinal > &indices) const;
 
 #ifdef HAVE_XPETRA_KOKKOS_REFACTOR
-    /// \brief Access the local KokkosSparse::StaticCrsGraph data
-    local_graph_type getLocalGraph () const;
```

Comment:
We do not keep this one within a `TPETRA_ENBALE_DEPRECATED_CODE` scope like we do for the CrsMatrix?

Reply:
Because this wasn't used anywhere in the code so I could get rid of it, I can add it and have it return device graph but it isn't tested or used

Reply:
@lucbv you o.k. with the way it is?

Reply:
Yes, I am fine with it, I actually think it's good to remove the old mechanics when it is not needed.
I have reviewed all the changes and I think this can be merged, let me ask quickly the rest of the team before I press the merge button.